sdk/communication/communication-chat/src/generated/src/chatApiClientContext.ts

```diff
@@ -10,7 +10,7 @@ import * as coreHttp from "@azure/core-http";
 import { ChatApiClientOptionalParams } from "./models";
 
 const packageName = "azure-communication-chat";
-const packageVersion = "1.1.0-beta.1";
+const packageVersion = "1.0.0";
```

Comment:
I'm not sure why this package version changed. It seems like it was incorrectly manually changed previously.